<launch>
	<node pkg="nanook" type="motor_driver.py" name="nanook_motor_driver">
		<param name="serial_port" type="string" value="/dev/ttyACM0" />
		<param name="baud_rate" type="int" value="9600" />
	</node>
</launch>
